db/write_batch.cc

```diff
@@ -839,7 +810,7 @@ Status WriteBatchInternal::Put(WriteBatch* b, uint32_t column_family_id,
     // inserted into memtable.
     b->prot_info_->entries_.emplace_back(
         ProtectionInfo64()
-            .ProtectKVOT(key, value, kTypeValue, timestamp)
+            .ProtectKVOT(key, value, kTypeValue, "")
```

Comment:
I guess we could eliminate this parameter if it's always empty, right?

Reply:
One more thing: I think the `T` in `KVOT` stands for timestamp; I think we could consider renaming this to `ProtectKVO` now that the timestamp is gone.

Reply:
THinking about it too. Already cc'ed @ajkr for his opinion. I am thinking using a different PR for the renaming of `kv_checksum`.